src/wavm/openmp.cpp

```diff
@@ -22,14 +22,15 @@ namespace wasm {
 // ------------------------------------------------
 
 #define OMP_FUNC(str)                                                          \
-    auto ctx = threads::getOpenMPContext();                                    \
+    threads::OpenMPContext& ctx = threads::getOpenMPContext();                 \
```

Comment:
This file contains the changes to run OpenMP thread `0` in the "main" thread for this module (which will obviously be a child thread of the main Faasm runtime thread)